srv/src/main/resources/application.yaml

```diff
@@ -109,3 +109,24 @@ spring:
 cds:
   data-source:
     auto-config.enabled: false
+---
+spring:
+  config.activate.on-profile: ft
+cds:
+  model:
+    provider:
+      url: http://localhost:4004
+  datasource:
+    csv-initialization-mode: "always"
+  security:
+    defaultRestrictionLevel: "any"
+    mock.users:
+      - name: carol
+        features:
+          - isbn
+      - name: erin
+        features:
+          - isbn
+          - discount
```

Comment:
as the FeatureTogglesProvider  toggles the features by role I think we should rather assign roles to the users and not features

Reply:
We advertise this on Capire. https://cap.cloud.sap/docs/guides/extensibility/feature-toggles?q=feature+toggles#in-development

But I do not mind either way.